src/align.cc

```diff
@@ -51,7 +51,7 @@ NumericVector c_impute(NumericVector x, NumericVector y, NumericVector rowname)
 
   int j = 0;
   for (int i = 0; i < N; ++i) {
-    if (rowname[i] == y[j] && j < L) {
+    if (j < L && rowname[i] == y[j]) {
```

Comment:
Is this one of bugs/issues we had on CRAN?

Reply:
Yes, this was the cause of the invalid read error